services/new.go

```diff
@@ -167,8 +167,12 @@ func parseConnectionString(ConnStr string) (ty string, ps []types.Pair, err erro
 	for _, v := range strings.Split(rest, "&") {
 		opt := strings.SplitN(v, "=", 2)
 		if len(opt) != 2 {
-			// && or &key&, ignore
-			continue
+			if v != "" {
+				opt = append(opt, v, "true")
+			} else {
+				// && , ignore
+				continue
+			}
```

Comment:
It's hard-coded here.
How about not generate pairs for `enable_xxx` features and register parsing functions for `StorageFeatures` and `ServiceFeatures`? Then we can collection pairs without values for `StorageFeatures` or `ServiceFeatures` parsing?

Reply:
Current implementation LGTM, we will refactor this part later, so don't be worried.

Reply:
Got it.